profiles/base.yaml

```diff
@@ -114,7 +114,8 @@ empty_hunting_room_messages:
 - Anyone here?
 hunting_buddies:
 stop_hunting_if_bleeding: true
-
+pre_hunt_buff: false
```

Comment:
If I change and include the prebuff waggle set in the check, this doesn't need the boolean.